docs/site-planning.md

```diff
@@ -1,44 +1,86 @@
+administrator, your task is to make your existing computing and storage resources available to and reliable for your
+supported VOs.
 
-Depending on the VOs you want to support, data services may not be necessary.  Even a compute element is not strictly
-necessary: OSG offers a "hosted CE" service where OSG will run the software provided only with a SSH connection to the
-batch submit host.  Contact <mailto:help@opensciencegrid.org> for more information on the hosted CE.
+We break this down into three tasks:
 
-- The simplest way to support OSG is to only provide SSH logins for the hosted CE and install the worker node
-  environment.
-- Larger or more complex sites will elect to run their own CE in addition to the WN environment and the simplest
-  data service (HTTP proxy cache).
-- The largest sites will additionally run large-scale data services such as a "storage element".  This is often required
-  for sites that want to support more complex organizations such as ATLAS or CMS.
+- Getting "pilot jobs" submitted to your site batch system.
+- Establishing a OSG runtime environment for running jobs.
+- Delivering data to payload applications to be processed.
+
+There are multiple approaches for each item, depending on the VOs you support and time you have to invest on the OSG.
 
 !!! note
     An essential concept on the OSG is the "pilot job".  The pilot, which arrives at your batch system, is sent by the
     VO and gets a resource allocation.  However, it _does not_ contain any scientific payload.  Once started, it will
     connect back to a resource pool and pull down individuals' scientific "payload jobs".  Hence, we do not think about
     submitting "jobs" to sites but rather "resource requests".
 
+### Pilot Jobs
+
+Traditionally, an OSG *Compute Element* (CE) provides remote access for VOs to submit pilot jobs to your local batch
+system.  However, today, there are two options for accepting pilot jobs at your site:
+
+- **Hosted CE**: OSG will run and operate the CE services; the site only needs to provide a SSH pubkey-based
+   authentication access to the central OSG host.  OSG will interface with the VO and submit pilots directly to your
+   batch system via SSH.  By far, this is the _simplest option_: however, it is less-scalable and the site delegates
+   many of the scheduling decisions to the OSG.  Contact <mailto:help@opensciencegrid.org> for more information on the
+   hosted CE.
+- **OSG CE**: The traditional option where the site installs and operates a HTCondor-based CE on a dedicated host.
+   This provides the best scalability and flexibility, but may require an ongoing time investment from the site.  The
+   OSG CE install and operation is covered in this documentation page.
+
+### Runtime environment
+
+The OSG provides a very minimal runtime environment that can be deployed via tarball, RPM, or through a global
```

Comment:
I think we should indicate that this means installing software on a site's worker nodes